changelogs/fragments/5241-homebrew-add-linux-path.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - homebrew, homebrew_tap - added Homebrew on Linux path to defaults (https://github.com/ansible-collections/community.general/pull/5241).
```

Comment:
Please break it down in two lines, on for each module. The change description can be exactly the same.

Reply:
I think it's also fine to have both in one line. We definitely did that before.